upstream-community-operators/mongodb-enterprise/1.5.1/mongodb-enterprise.v1.5.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,383 @@
+              "configMapRef": {
+                "name": "my-project"
+              }
+            },
+            "type": "ReplicaSet",
+            "version": "4.2.2"
+          }
+        },
+        {
+          "apiVersion": "mongodb.com/v1",
+          "kind": "MongoDB",
+          "metadata": {
+            "name": "sample-sharded-cluster"
+          },
+          "spec": {
+            "configServerCount": 3,
+            "credentials": "my-credentials",
+            "mongodsPerShardCount": 3,
+            "mongosCount": 2,
+            "persistent": false,
+            "opsManager": {
+              "configMapRef": {
+                "name": "my-project"
+              }
+            },
+            "shardCount": 1,
+            "type": "ShardedCluster",
+            "version": "4.2.2"
+          }
+        },
+        {
+          "apiVersion": "mongodb.com/v1",
+          "kind": "MongoDBUser",
+          "metadata": {
+            "name": "my-replica-set-x509-user"
+          },
+          "spec": {
+            "db": "$external",
+            "mongodbResourceRef": {
+              "name": "my-replica-set"
+            },
+            "roles": [
+              {
+                "db": "admin",
+                "name": "dbOwner"
+              }
+            ],
+            "username": "CN=my-replica-set-x509-user,OU=cloud,O=MongoDB,L=New York,ST=New York,C=US"
+          }
+        },
+        {
+          "apiVersion": "mongodb.com/v1",
+          "kind": "MongoDBOpsManager",
+          "metadata": {
+            "name": "ops-manager"
+          },
+          "spec": {
+            "version": "4.2.8",
+            "adminCredentials": "ops-manager-admin",
+            "configuration": {
+              "mms.fromEmailAddr": "admin@thecompany.com"
+            },
+            "externalConnectivity": {
+              "type": "LoadBalancer"
+            },
+            "applicationDatabase": {
+              "members": 3,
+              "version": "4.2.2-ent",
+              "persistent": true,
+              "podSpec": {
+                "cpu": "0.25"
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: Database
+    certified: 'true'
+    containerImage: quay.io/mongodb/mongodb-enterprise-operator:1.5.1
+    createdAt: 2020-04-30T10:00:00Z
+    description: The MongoDB Enterprise Kubernetes Operator enables easy deploys of
+      MongoDB into Kubernetes clusters, using our management, monitoring and backup
+      platforms, Ops Manager and Cloud Manager.
+    repository: https://github.com/mongodb/mongodb-enterprise-kubernetes
+    support: MongoDB Community Support
+  name: mongodb-enterprise.v1.5.1
+  namespace: placeholder
+spec:
+  customresourcedefinitions:
+    owned:
+    - description: MongoDB Deployment
+      displayName: MongoDB Deployment
+      group: mongodb.com
+      kind: MongoDB
+      name: mongodb.mongodb.com
+      version: v1
+      resources:
```

Comment:
Can you please confirm that this is the correct syntax for ownership of resources for operands?

Reply:
Yes, this is correct.